src/app/shared/services/workshops/app-workshop/app-workshops.service.ts

```diff
@@ -93,6 +93,10 @@ export class AppWorkshopsService {
       params = params.set('OrderByField', filters.order);
     }
 
+    if (!!filters.statuses.length) {
```

Comment:
just small note 
on line 96 we write 
`if (!!filters.statuses.length) {`
on line 100
`if (filters.directions.length > 0) {`
note very consistent, but should work anyway.